src/components/List/ListItem/ListItem.tsx

```diff
@@ -53,8 +53,10 @@ export const ListItem = forwardRef<HTMLLIElement, Props>(
             <li
                 {...liProps}
                 ref={ref}
-                role="presentation"
+                role="option"
                 onClick={onClick}
+                aria-selected={isHighlighted}
+                onKeyDown={onClick}
```

Comment:
I think this isn't the best practice to call `onClick` for any key press. Normally it is only `Enter` that should trigger it. Can you make it a handle function instead, that calls the callback only in that case?

Reply:
I've pushed changes for this action could you please check it

Reply:
I had a quick look, and it looks ok. however, as mentioned in the other comment, I'm still looking into it. My concern is that a ListItem that doesn't have an `onClick` handles now still looks like an interactive element with `onKeyDown` and `role=option`. I'm trying to figure out what is the best way to go about it. Can you leave this PR for me for a bit until I figure these out?

Reply:
Sure